chemprop/features/features_generators.py

```diff
@@ -50,84 +50,131 @@ def get_available_features_generators() -> List[str]:
 
 
 @register_features_generator('morgan')
-def morgan_binary_features_generator(mol: Molecule,
+def morgan_binary_features_generator(mol_data: Union[Molecule, List[List[Molecule]]],
                                      radius: int = MORGAN_RADIUS,
                                      num_bits: int = MORGAN_NUM_BITS) -> np.ndarray:
     """
     Generates a binary Morgan fingerprint for a molecule.
-
-    :param mol: A molecule (i.e., either a SMILES or an RDKit molecule).
+    :param mol_data: A molecule (i.e., either a SMILES or an RDKit molecule).
     :param radius: Morgan fingerprint radius.
     :param num_bits: Number of bits in Morgan fingerprint.
     :return: A 1D numpy array containing the binary Morgan fingerprint.
     """
-    mol = Chem.MolFromSmiles(mol) if type(mol) == str else mol
-    features_vec = AllChem.GetMorganFingerprintAsBitVect(mol, radius, nBits=num_bits)
-    features = np.zeros((1,))
-    DataStructs.ConvertToNumpyArray(features_vec, features)
+
+    def fingerprint_single_molecule(m: Molecule) -> np.ndarray:
+        m = Chem.MolFromSmiles(m) if type(m) == str else m
+        features_vec = AllChem.GetMorganFingerprintAsBitVect(m, radius, nBits=num_bits)
+        single_mol_features = np.zeros((1,))
+        DataStructs.ConvertToNumpyArray(features_vec, single_mol_features)
+        return single_mol_features
+
+    if type(mol_data) == list:
+        features = []
+        for datapoint in mol_data:
+            entry_features = [fingerprint_single_molecule(molecule) for molecule in datapoint]
+            features.extend(entry_features)
```

Comment:
```suggestion
    if isinstance(mol_data, Iterable):
        features = [[fingerprint_single_molecule(mol) for mol in datum] for datum in mol_data]
```
applies to the rest as well